delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/model/ResourceEvent.scala

```diff
@@ -294,12 +305,13 @@ object ResourceEvent {
         ProjectScopedMetric.from(
           event,
           event match {
-            case _: ResourceCreated    => Created
-            case _: ResourceUpdated    => Updated
-            case _: ResourceRefreshed  => Refreshed
-            case _: ResourceTagAdded   => Tagged
-            case _: ResourceTagDeleted => TagDeleted
-            case _: ResourceDeprecated => Deprecated
+            case _: ResourceCreated       => Created
+            case _: ResourceUpdated       => Updated
+            case _: ResourceRefreshed     => Refreshed
+            case _: ResourceTagAdded      => Tagged
+            case _: ResourceTagDeleted    => TagDeleted
+            case _: ResourceDeprecated    => Deprecated
+            case _: ResourceSchemaUpdated => Updated
```

Comment:
This mapping felt correct to me

Reply:
Not sure but with the last changes (including tagging at creation for example), we will have to rework this part.